<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat_window</class>
 <widget class="QWidget" name="chat_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="friendNameText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>541</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="msgSendButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>500</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
